src/gui/layout/qgslayoutmanualtablewidget.cpp

```diff
@@ -170,106 +170,117 @@ bool QgsLayoutManualTableWidget::setNewItem( QgsLayoutItem *item )
   return true;
 }
 
-void QgsLayoutManualTableWidget::setTableContents()
+QgsTableEditorDialog *QgsLayoutManualTableWidget::openTableDesigner( QgsLayoutFrame *frame, QWidget *parent )
```

Comment:
Looks like this is opening a new dialog with every double click. I'd have thought it makes more sense to keep the same behavior as setTableContents had for a double-click, where the existing dialog is just focused. Otherwise multiple double-clicks will end up with many dialogs opened...

Reply:
Ok, I'll change that.